about.html

```diff
@@ -1,73 +1,136 @@
+    <link rel="stylesheet" href="css/about.css">
 
 </head>
 
 <body>
     <div class="container about-page">
-     
-    About
 
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
+        <nav class="navigation">
+            <a href="#">Home</a>
+            <a href="about.html">About</a>
+            <a href="#">Products</a>
+            <a href="#">Blog</a>
+            <a href="#">Contact</a>
 
+            <a href="index.html"><img class="logo" src="img/lambda-black.png" alt="lambda-logo"></a>
 
-    <img src="img/about-plan.png" alt="strategy">
-    
-    Strategy
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
+        </nav>
 
-    Learn More
+        <img class="jumbo-about" src="img/jumbo-about.png" alt="jumbo-about"></img>
 
+        <div class="text-container">
+            <h2>About</h2>
+            <p>Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of
+                disruptive innovation via workplace diversity and empowerment.</p>
 
+        </div>
+        <hr>
 
-    <img src="img/about-working.png" alt="strategy">
-    
-    How We Work
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
+        <section class="about-top">
 
-    Learn More
+            <div class="strategy-top">
+                <img class="plan" src="img/about-plan.png" alt="strategy"></img>
 
+                <section class="strategy">
+                    <h3>Strategy</h3>
+                </section>
 
+                <section class="p-strategy">
+                    <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
+                </section>
 
-    <img src="img/about-office.png" alt="strategy">
-    
-    Places We Work
+                <div class="button1">
+                    <button>Learn More</button>
+                </div>
+            </div>
 
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
+            <div class="working-top">
 
-    Learn More
+                <img class="working" src="img/about-working.png" alt="strategy">
 
-    <img src="img/about-meeting.png" alt="strategy">
-    
-    Collaboration
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
+                <section class="work">
```

Comment:
Same goes here - you don't need to wrap these in section tags.

Reply:
Couldn't fix this